std/path/glob.ts

```diff
@@ -1,5 +1,3 @@
-// globToRegExp() is originall ported from globrex@0.1.2.
-// Copyright 2018 Terkel Gjervig Nielsen. All rights reserved. MIT license.
```

Comment:
This is being removed because this is a new implementation?

Reply:
Yes, this one necessarily rewrites the whole thing to achieve:
```
- feat: Match characters literally on segment parse failure
- fix: Contain any glob syntax to its path segment
```